sklearn/calibration.py

```diff
@@ -141,6 +143,12 @@ class CalibratedClassifierCV(ClassifierMixin,
     classes_ : ndarray of shape (n_classes,)
         The class labels.
 
+    n_features_in_ : int
+        Number of features seen during :term:`fit`. Only defined if the
+        underlying base_estimator exposes such an attribute when fit.
+
+        .. versionadded:: 0.24
```

Comment:
```suggestion
        .. versionadded:: 1.0
```

Reply:
since it was there, I think it's more accurate to document that it was introduced in 0.24, even if not documented.